lib/ood_packaging/build.rb

```diff
@@ -166,7 +166,8 @@ def bootstrap_rpm!
     bootstrap_gpg! if gpg_sign?
     if podman_runtime?
       puts "\tBootstrap /root".blue
-      sh "cp -r #{ctr_rpmmacros} #{ctr_gpg_dir} /root/"
+      sh "cp -r #{ctr_rpmmacros} /root/"
+      sh "cp -r #{ctr_gpg_dir} /root/" if gpg_sign?
```

Comment:
See #234 So you're logic is correct but also needs `&& build_box.dnf?`

```suggestion
      sh "cp -r #{ctr_gpg_dir} /root/" if gpg_sign? && build_box.dnf?
```

Reply:
The /home/ood/.gnupg will only be boostrapped in container for DNF OSes.